x-pack/plugin/sql/src/main/java/org/elasticsearch/xpack/sql/querydsl/agg/GroupByKey.java

```diff
@@ -62,6 +62,7 @@ public final CompositeValuesSourceBuilder<?> asValueSource() {
             } else if (script.outputType() == IP) {
                 builder.userValuetypeHint(ValueType.IP);
             }
+            // TODO hint for VERSION
```

Comment:
?

Reply:
I see `ValueType` is  deprecated, so I was unsure if adding further content to that class would be acceptable and if there could be an alternative. Any suggestions?

Reply:
After off-line discussions and investigations, it seems it's not needed.